release_tools/README.md

```diff
@@ -229,7 +131,8 @@ Fedora builds
 -   and [Package Update How
     To](https://fedoraproject.org/wiki/Package_update_HOWTO)
 
-Copr builds
+
+COPR Builds
```

Comment:
I wonder if  we should leave the documentation here since we no longer maintain the copr repo for SSG.

Reply:
Got rid of it